spec/order_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'order'
+
+describe Order do
+  let(:small_order){1}
+  let(:large_order){5}
+  let(:messager){double(:messager, send_sms: true)}
+  let(:messager_klass){double(:messager_klass, :new => messager)}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)